src/pages/_app.tsx

```diff
@@ -1,12 +1,19 @@
 import Layout from '@/components/Layout';
 import '@/styles/globals.css';
 import type { AppProps } from 'next/app';
+import Script from 'next/script';
 
 export default function App({ Component, pageProps }: AppProps) {
 	return (
 		<>
 			<Layout>
 				<Component {...pageProps} />
+				{/* Beam Analytics */}
+				<Script
+					src="https://beamanalytics.b-cdn.net/beam.min.js"
+					data-token={process.env.BEAM_DATA_TOKEN}
+					async
+				/>
```

Comment:
This script is already used in the `Header` component. Is it okay to duplicate it here?